.gitignore

```diff
@@ -11,3 +11,4 @@ MANIFEST
 keel/.DS_Store
 env/
 *db.sqlite3
+/var
```

Comment:
/var directory should be the system directory of a unix based system ......where all the logs are saved
/var/log
it shouldn't be in project's root.
so it wont be needed in gitignore

Reply:
done